tests/async_tests/test_interfaces.py

```diff
@@ -27,11 +29,11 @@
         assert http_version == b"HTTP/1.1"
         assert status_code == 200
         assert reason == b"OK"
-        assert len(http._connections[url[:3]]) == 1
```

Comment:
It's slightly awkward to add all these `type: ignore` but I thought it was better than exposing `connections`.

Reply:
Agreed, `# type: ignore` are fine here IMO. :)